web/frontend/src/components/slam-2d-render.vue

```diff
@@ -42,27 +52,100 @@ scene.add(camera);
 const controls = new MapControls(camera, canvas);
 controls.enableRotate = false;
 
-const raycaster = new MouseRaycaster({ camera, renderer, recursive: false });
+const raycaster = new MouseRaycaster({ camera, canvas, recursive: false });
```

Comment:
`MouseRaycaster` changed in the latest version of `trzy`: renderer should be passed as a constructor arg and `raycaster.addEventListener()` below should be replaced with `raycaster.on()`